charts/spire/charts/spire-server/values.yaml

```diff
@@ -209,6 +209,7 @@ upstreamAuthority:
       renewBefore: ""
   spire:
     enabled: false
+    upstreamDriver: upstream.csi.spiffe.io
```

Comment:
Here's the value I would expect there to be no default value.  That would mean it would not exist in the values.yaml file here; but a _helpers.tpl would check it exists when the "enabled" flag is set to true.

This forces the value to be provided in a values.yaml file manually, or via the cli manually, preventing any kind of "default" value.

Reply:
Done